src/data/complex/module.lean

```diff
@@ -181,6 +181,11 @@ restrict_scalars.is_scalar_tower ℝ ℂ E
   (x : ℂ) • y = x • y :=
 rfl
 
+@[priority 900]
+instance module.real_complex_smul_comm_class {E F : Type*}
```

Comment:
I would call it `smul_comm_class.complex_to_real`. Also, I would use `α` or `M` instead of `E`. Otherwise LGTM. Thanks!
bors d+

Reply:
Possibly worth stating in the docstring that this is specifically about `module.complex_to_real`